client/app/lib/constants/providerSpecs.coffee

```diff
@@ -0,0 +1,20 @@
+module.exports =
```

Comment:
👍 So @cansirin we need to move this file under `client/app/lib/util`, make it a function that accepts a `provider` argument, and return the list depending on this `provider` argument. And rename it to `makeProviderStacks.coffee`